doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md

```diff
@@ -0,0 +1,245 @@
+The role is associated with a Kubernetes service account created in the namespace that you specify (your cluster's) and annotated with `eks.amazonaws.com/role-arn:arn:aws:iam::111122223333:role/my-role-name`.
+
+
+!!! Example TL;DR
+      ``` shell
+      eksctl create iamserviceaccount \
+      --name "<my-service-account>" \
+      --cluster "<my-cluster>" \
+      --attach-policy-arn \ "<Copy the arn of your policy HERE>" \
+      --approve \ 
+      --override-existing-serviceaccounts
+      ```
+
+
+## 5. Mount Your Secrets In Your EKS Cluster
+To show secrets in EKS as though they are files on the filesystem, you need to create a **SecretProviderClass** YAML file that contains information about your secrets as well as information on how to display them in the EKS pod. Use the file provided below and run `kubectl apply -f yoursecretclass.yaml`.
+
+The SecretProviderClass must be in the same namespace as the EKS cluster.
+
+```yaml
+metadata:
+  # Insert your secret name
+  name: pach-secrets
+spec:
+  provider: aws
+  parameters:
+    objects: |
+        - objectName: "pach-secrets"
+          objectType: "secretsmanager"
+          jmesPath:
+            - path: root_token
+              objectAlias: root-token
+            - path: postgresql_password
+              objectAlias: postgresql-password
+            - path: OAUTH_CLIENT_SECRET
+              objectAlias: OAUTH_CLIENT_SECRET
+            - path: enterprise_license_key
+              objectAlias: enterprise-license-key
+            - path: pachd_oauth_client_secret	
+              objectAlias: pachd-oauth-client-secret
+            - path: enterprise_secret
+              objectAlias: enterprise-secret
+  secretObjects:
+  - data:
+    - key: root-token
+      objectName: root-token
+    secretName: root-token
+    type: Opaque
+  - data:
+    - key: postgresql-password
+      objectName: postgresql-password
+    secretName: postgresql-password
+    type: Opaque
+  - data:
+    - key: OAUTH_CLIENT_SECRET
+      objectName: OAUTH_CLIENT_SECRET
+    secretName: console-oauth-client-secret
+    type: Opaque
+  - data:
+    - key: enterprise-license-key
+      objectName: enterprise-license-key
+    secretName: enterprise-license-key
+    type: Opaque
+  - data:
+    - key: pachd-oauth-client-secret	
+      objectName: pachd-oauth-client-secret	
+    secretName: pachd-oauth-client-secret
+    type: Opaque
+  - data:
+    - key: enterprise-secret
+      objectName: enterprise-secret
+    secretName: enterprise-secret
+    type: Opaque
+```
+
+## 6. Create A Syncer Pod
+
+Once your secret class is configured, a pod needs to request the class to trigger the CSI driver and retrieve the secrets in Kubernetes. 
+Update the file below with your `serviceAccountName` and  `secretProviderClass` before you run a `kubectl apply -f syncerpod.yaml`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-syncer
+spec:
+  containers:
+  - name: ubuntu
```

Comment:
This isn't ubuntu.

Reply:
See https://github.com/pachyderm/pachyderm/pull/7597